interfaces/builtin/fwupd.go

```diff
@@ -64,13 +68,78 @@ const fwupdPermanentSlotAppArmor = `
   /sys/firmware/efi/esrt/entries/** r,
 
   # Allow fwupd to access system information
-  /sys/devices/virtual/dmi/id/product_name r,
-  /sys/devices/virtual/dmi/id/sys_vendor r,
+  /sys/firmware/dmi/tables/smbios_entry_point r,
+  /sys/devices/virtual/dmi/id/{sys,bios,product,chassis,board}_{name,version,vendor,date,serial,sku,type,asset_tag,family} r,
+
+  /sys/firmware/dmi/tables/DMI r,
+  /sys/bus/usb/devices/ r,
+
```

Comment:
`/sys/bus/spi` for flashrom plugin

Reply:
I wonder what files need to be read here.

Also what paths will flashrom itself need to access?

Reply:
@hughsie, do you know?

Reply:
Ohh libflashrom is evil. /dev/mem for one. I'd be included to not support flashrom as it's doing iopl and awful things.

Reply:
I guess we should also explicitly disable it in the snap build then once it gets confined.